### ISSUE TITLE
3.7.0 Symbols went on holidays

### ISSUE THREAD

**Issue by PeterMozesMerl** [teamcapybara/capybara#2093]
## Code

```ruby
expect(page).to have_css :button
```

## Expected Behavior
The spec/test either succeeds or fails, depending on the content of the page, the rest of the spec, and a few other things.

### From ages ago to 3.6.0

Works like a charm.

### 3.7.0
```
TypeError:
       no implicit conversion of Symbol into String
```

## Wish

Although I’ve never used the find(:link) syntax (described in https://github.com/teamcapybara/capybara/issues/849), I can imagine the issues.

However, the first parameter of have_css is explicit. I see no wrong in converting the Symbol to String. The `have_css :button` is Rails-like, faster to type, and has a more relaxed color in my IDE than the single quoted string version.

Can we please get it back?

**Comment by twalpole** [comment]
Please try the `sym_to_string` branch and see if that fixes your issue

**Comment by PeterMozesMerl** [comment]
Thank you. I used the branch with a project that has 305 Capybara examples. All passed.

**Comment by twalpole** [comment]
Ok -- It's in the master and 3.7_stable branch now -- Assuming no other issues with 3.7.0 come up I'll probably release 3.7.1, including this change, later today

**Comment by twalpole** [comment]
Fixed via edc8a25f0bab4d2703b69922ce36a2df02669bdc

**Comment by PeterMozesMerl** [comment]
I understand it wasn’t officially supported. Thank you again.

**Comment by twalpole** [comment]
Released in 3.7.1
